server.py

```diff
@@ -550,6 +554,42 @@ def get(self):
 
         return return_dict
 
+class PendingMergesResource(restful.Resource):
+    def get(self):
+        pass
```

Comment:
i'm not sure. if the "review merges" stuff is going to be a real web-interface, then yes, we'd need this endpoint. OTOH, if you want to do it more as a script, then we don't really need this.

what do you think?